homeassistant/components/recorder/models.py

```diff
@@ -361,6 +363,26 @@ def __repr__(self) -> str:
         )
 
 
+class StatisticsRuns(Base):  # type: ignore
```

Comment:
We will need a migration to create entries for this

Reply:
Fixed in 865b04c